compiler/plugins/input/StableHLO/Conversion/test/vhlo_stablehlo_mix_invalid.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: iree-opt --iree-check-vhlostablehlo-mix-usage --split-input-file %s --verify-diagnostics
+
+vhlo.func_v1 @vhlo_stablehlo_func(%arg0: !vhlo.tensor_v1<!vhlo.i32_v1>) -> (!vhlo.tensor_v1<!vhlo.i32_v1>) {
+  // expected-error @+1 {{Using VHLO and StableHLO Ops in the same module is not supported}}
+  %0 = "vhlo.constant_v1"() <{value = #vhlo.tensor_v1<dense<1> : tensor<i32>>}> : () -> !vhlo.tensor_v1<!vhlo.i32_v1>
+  %1 = builtin.unrealized_conversion_cast %0 : !vhlo.tensor_v1<!vhlo.i32_v1> to tensor<i32>
+  // expected-remark @+1 {{Last Stablehlo Op was found here}}
```

Comment:
These `expected-` strings probably need updating after the text changes

Reply:
Fixed: 🤦🏽 sorry!